test.py

```diff
@@ -1,4 +1,3 @@
 from test1 import func
 
-
```

Comment:
Removing the function call without an explanation or replacement makes the test file less useful. Is this a temporary change or is the test being rewritten? Consider adding a comment explaining the purpose of this change.

```suggestion
Add a comment explaining why `func()` was removed.
```